src/metadata.rs

```diff
@@ -190,7 +190,7 @@ impl IntoIterator for Metadata {
 
 // Disable implicit_hasher; suggested code fix does not compile. I think this might be a false
 // positive, but I'm not sure.
-#[cfg_attr(feature = "cargo-clippy", allow(clippy::implicit_hasher))]
+#[cfg_attr(clippy, allow(clippy::implicit_hasher))]
```

Comment:
Seems like it would be enough with just `#[allow(clippy::implicit_hasher)]`, according to the linked article. :-)